src/maintainer/knowledge_base.rst

```diff
@@ -202,27 +234,22 @@ To skip building with a particular ``vc`` version, add a skip statement.
       build:
         - {{ compiler('cxx') }}
 
-Using vs2019
--------------
-
-To use ``vs2019`` make the following changes:
+Using vs2022
+^^^^^^^^^^^^
 
-In ``conda_build_config.yaml`` file:
+In ``recipe/conda_build_config.yaml`` file:
 
 .. code-block:: yaml
 
     c_compiler:    # [win]
-    - vs2019       # [win]
+    - vs2022       # [win]
     cxx_compiler:  # [win]
-    - vs2019       # [win]
-
+    - vs2022       # [win]
```

Comment:
We need to account for the [new `arm64` architecture in Windows](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/360bec574e3c93156269d6475b2c5c46be57e304/recipe/conda_build_config.yaml#L4-L5), so the selector needs an update.

Reply:
I disagree here. The current pinning _has_ to split x86_64 and arm64 by necessity, but we don't. In particular, specifying `vs2022  # [win]` does nothing on the arm64-side.

I would argue that it's _better_ for a feedstock to be using a single compiler version per platform whenever possible[^1], and certainly the docs should reflect that. It's also plain simpler to read & understand.

[^1]: but the introduction of a new platform forces us to accept that, at least on conda-forge level

Reply:
Sounds like we should revert 7cc08a7?

Reply:
Yes please

Reply:
7af43c12